dateparser/data/date_translation_data/en.py

```diff
@@ -252,6 +259,14 @@
             "(\\d+) second ago",
             "(\\d+) seconds ago",
             "(\\d+) sec ago"
+        ],
```

Comment:
don't forget to update this file with `write_complete_data.py` when updating the `yaml` files.